curriculum/challenges/english/01-responsive-web-design/css-piano/part-018.md

```diff
@@ -0,0 +1,115 @@
+---
+id: 612ea4c4993aba52ab4aa32e
+title: Part 18
+challengeType: 0
+dashedName: part-18
+---
+
+# --description--
+
+Now it is time to use the pseudo-selectors you prepared for earlier. To create the black keys, add a new `.key.black--key::after` selector. This will target the elements with the class `key black--key`, and select the pseudo-element after these elements in the HTML.
+
+In the new selector, set the `background-color` to `#1d1e22`. Also set the `content` property to `""`. This will make the pseudo-elements empty.
```

Comment:
Similar to above: This is the first time `content` is being introduced. What is the `content` property?